downstream/modules/platform/proc-controller-select-capacity.adoc

```diff
@@ -0,0 +1,30 @@
+[id="controller-select-capacity"]
+
+= Selecting the right capacity
+
+Selecting a capacity out of the CPU-bound or the memory-bound capacity limits is selecting between the minimum or maximum number of forks.
+In the previous examples, the CPU capacity permits a maximum of 16 forks while the memory capacity permits 20. 
+For some systems, the disparity between these can be large and you might want to have a balance between these two.
+
+The instance field `capacity_adjustment` enables you to select how much of one or the other you want to consider. 
```

Comment:
The capacity_adjustment instance field 

how much of ? or ? you want to consider (mem_capacity, cpu_capacity)?